router/router.go

```diff
@@ -31,12 +37,22 @@ func Create(db *database.GormDatabase, vInfo *model.VersionInfo, conf *config.Co
 		DB:       db,
 		ImageDir: conf.UploadedImagesDir,
 	}
-	userHandler := api.UserAPI{DB: db, PasswordStrength: conf.PassStrength, NotifyDeleted: streamHandler.NotifyDeletedUser}
+	userChangeNotifier := new(api.UserChangeNotifier)
+	userHandler := api.UserAPI{DB: db, PasswordStrength: conf.PassStrength, UserChangeNotifier: userChangeNotifier}
 
-	g := gin.New()
+	pluginManager, err := plugin.NewManager(db, conf.PluginsDir, g.Group("/plugin/:id/custom/"), streamHandler)
```

Comment:
What do you think it the best? 

I'd be okay with using a different root path but I'd prefer using `/custom/P000000000` because the plugins can do pretty much everything, it doesn't have to be a hook. 
The last option seems also good.